suzieq/poller/worker/services/service.py

```diff
@@ -22,7 +22,10 @@
 from suzieq.shared.utils import known_devtypes
 from suzieq.version import SUZIEQ_VERSION
 
-HOLD_TIME_IN_MSECS = 60000  # How long b4 declaring node dead
+# How long b4 declaring node dead
+HOLD_TIME_IN_MSECS = 60000
+# How many unsuccessful polls before marking records with active=false
+HYSTERESIS_FAILURE_CNT = 2
```

Comment:
Can you make this three please? Why 3? I think just about every protocol I know has a dead time thats 3 times the keep alive time :)

Reply:
I thought one would be too few but no specific reason for choosing 2. I'll make it 3.